### ISSUE TITLE
removing lang attribute deletes the whole file content

### ISSUE THREAD

**Issue by Olyno** [inlang/inlang#1076]
## Problem

In Svelet kit projects, when we install inlang, it wants to remove the "lang" attribute from the "html" file, but instead deletes the entire file.

### Expected behavior

This should delete only the attribute, not the entire file. Is it even worth modifying the attribute?

## Steps to reproduce

1. Install the svelte kit sdk for inlang
2. Run your dev server
3. Tada

### Additional information

- `inlang.config.js` file:

```js
/**
 * @type { import("@inlang/core/config").DefineConfig }
 */
export async function defineConfig(env) {
	const { default: jsonPlugin } = await env.$import(
		'https://cdn.jsdelivr.net/npm/@inlang/plugin-json@3/dist/index.js'
	);
	const { default: sdkPlugin } = await env.$import(
		'https://cdn.jsdelivr.net/npm/@inlang/sdk-js-plugin@0.7.1/dist/index.js'
	);

	return {
		referenceLanguage: 'en',
		plugins: [
			jsonPlugin({
				pathPattern: './i18n/{language}.json'
			}),
			sdkPlugin({
				languageNegotiation: {
					strategies: [{ type: 'localStorage' }]
				}
			})
		]
	};
}
```

- `npx @inlang/cli config validate` output:

```sh
ℹ 🔎 Validating the config file...                                                                  00:50:43
✔ 🎉 Inlang config is valid!                                                                        00:50:43
```

- Browser: Firefox
- Operating system: Deepin OS
- Screenshots: /

**Comment by ivanhofer** [comment]
Could you please share the contents of that file?

**Comment by Olyno** [comment]
Sure! Here is the file i'm using:

```html
<!DOCTYPE html>
<html lang="en">
        <head>
                <meta charset="utf-8" />
                <link rel="icon" href="%sveltekit.assets%/favicon.png" />
                <meta name="viewport" content="width=device-width" />
                <script
                        async
                        src="https://analytics.com/script.js"
                        data-website-id="c46bbedd-b260-4c3b-a238-a0381743f6f4"
                        data-domains="analytics.com"
                ></script>
                %sveltekit.head%
        </head>
        <body data-sveltekit-preload-data="hover">
                <div style="display: contents">%sveltekit.body%</div>
        </body>
</html>
```

**Comment by ivanhofer** [comment]
The issue should be fixed in version `0.8.0`
